tests/baselines/reference/tscWatch/emitAndErrorUpdates/default/transitive-exports/no-circular-import/export.js

```diff
@@ -444,6 +450,8 @@ FsWatches::
   {}
 
 FsWatchesRecursive::
+/user/username/projects/myproject/lib1:

```

Comment:
Just as an FYI, all these extra watchers are here because these tests have an import that looks like `"./tools/tools.interface"` which now looks for a `./tools/tools.d.interface.ts` (for the potential literal `tools.interface` file) before adding a `.js` or `.ts` extension onto the import to find what it currently finds. The additional failed lookup location triggers adding a directory watcher, since now a new file may appear which changes the resolution result~